### PR TITLE
feat: implement `get_imports()` API method

### PR DESCRIPTION
**Pull request by unparalleled-js**
### What I did

fixes: #43 
Fixes: APE-406
This is needed for Etherscan verification and I think also helps Ape core compile Vyper projects (IICR why we did it in `ape-solidity`).

This is step 1 to getting Verification working for Vyper contracts.

### Checklist

- [x] Passes all linting checks (pre-commit and CI jobs)
- [x] New test cases have been added and are passing
- [ ] Documentation has been updated
- [x] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

**Comment by unparalleled-js** [comment]
- [ ] handle `from contract import foo` style

**Comment by NotPeopling2day** [comment]
@fubuloubu can give some more detail, but there's at least a third way for Vyper imports, using .json files. We'll want to account for in the tests as well.

1. import ...
2. from ... import
3. json file

https://vyper.readthedocs.io/en/v0.2.11/compiling-a-contract.html#vyper-json

File: ape_vyper/compiler.py [hunk 1]
```diff
@@ -61,6 +62,42 @@ def name(self) -> str:
     def evm_version(self) -> Optional[str]:
         return self.config.evm_version
 
+    def get_imports(  # type: ignore[empty-body]
```
Comment (fubuloubu) [inline ape_vyper/compiler.py:65]: Why is this type ignore needed?

**Review by fubuloubu — APPROVED** [review]

**Comment by fubuloubu** [comment]
> @fubuloubu can give some more detail, but there's at least a third way for Vyper imports, using .json files. We'll want to account for in the tests as well.
> 
> 1. import ...
> 2. from ... import
> 3. json file
> 
> https://vyper.readthedocs.io/en/v0.2.11/compiling-a-contract.html#vyper-json

Yes, you can pull in ABI JSON and ethPM Manifest files this way

**Comment by unparalleled-js** [comment]
> @fubuloubu can give some more detail, but there's at least a third way for Vyper imports, using .json files. We'll want to account for in the tests as well.
> 
> 1. import ...
> 2. from ... import
> 3. json file
> 
> https://vyper.readthedocs.io/en/v0.2.11/compiling-a-contract.html#vyper-json

That sounds good.. but maybe a little complex enough to warrant its own PR?

**Comment by fubuloubu** [comment]
> > @fubuloubu can give some more detail, but there's at least a third way for Vyper imports, using .json files. We'll want to account for in the tests as well.
> > 
> > 1. import ...
> > 2. from ... import
> > 3. json file
> > 
> > https://vyper.readthedocs.io/en/v0.2.11/compiling-a-contract.html#vyper-json
> 
> That sounds good.. but maybe a little complex enough to warrant its own PR?

Agreed

**Review by NotPeopling2day — DISMISSED** [review]

File: ape_vyper/compiler.py (hunk 1 above)
Comment (unparalleled-js) [reply ape_vyper/compiler.py]: fixed!

**Review by fubuloubu — APPROVED** [review]

**Pull request merged**